src/const/chapters.ts

```diff
@@ -399,6 +399,36 @@ export const chapters: IChapter[] = [
       `
       },
     ]
+  },
+  {
+    chapterName: 'Box Model',
+    levels: [
+      {
+        levelName: 'border',
+        instructions: 'Observe these images - one is without a border, while the other has one. Make use of the <code>border</code> property to rectify the issue.',
```

Comment:
What about: 'Take a look at these images - one has no border, while the other has one. Utilize the <code>border</code> property to rectify the issue.'